docs/guides/extensions/modules-build-portal.md

```diff
@@ -0,0 +1,969 @@
+
+```js
+import useDirectusToken from './use-directus-token';
+```
+
+Include the function `AddTokenToURL` as a variable from the new script.
+
+```js
+setup(props) {
+	const router = useRouter();
+	const api = useApi();
+	const { addTokenToURL } = useDirectusToken(api);
+
+	// Existing code
+}
+```
+
+Then wrap any internal images with this function:
+
+```js
+page_banner.value = addTokenToURL(`/assets/${item.banner}?width=2000&height=563&fit=cover`);
+```
+
+:::info External Images
+
+If you are using images from external sources, the host must be added to the Content Security Policy (CSP) inside the
+environment or config file.
+
+:::
+
+## Style the Module
+
+Add some SCSS at the bottom of the `module.vue` file. When dealing with multiple vue files, don’t scope the SCSS,
+instead prefix each class with a unique reference to prevent changing other components in Directus. In this example, use
+the following SCSS:
+
+```html
+<style lang="scss">
+.lp-container {
+	padding: var(--content-padding);
+	padding-top: 0;
+	width: 100%;
+	max-width: 1024px;
+
+
+	&> div {
+		margin-bottom: var(--content-padding);
+	}
+}
+
+
+.lp-banner {
+	border-radius: var(--border-radius);
+	overflow: hidden;
+
+
+	img {
+		display: block;
+		width: 100%;
+	}
+}
+
+
+.lp-cards {
+	display: grid;
+	grid-template-columns: repeat(4, 1fr);
+	column-gap: var(--input-padding);
+	row-gap: var(--input-padding);
+
+
+	.lp-card {
+		display: flex;
+		flex-wrap: wrap;
+		align-items: center;
+		justify-content: center;
+		text-align: center;
+		border-radius: var(--border-radius);
+		padding: var(--input-padding);
+		color: white;
+
+
+		.v-icon {
+			width: 100%;
+			height: 50px;
+			margin-bottom: 6px;
+
+
+			i {
+				font-size: 50px;
+				color: white;
+			}
+		}
+
+
+		.lp-card-title {
+			display: block;
+			font-weight: bold;
+			font-size: 1.4em;
+			line-height: 1.2;
+		}
+	}
+}
+</style>
+```
+
+This will format the banner, cards and the container. It’s a good idea to make use of the native CSS of Directus as much
+as possible so your module appears part of Directus.
+
+Now the page will look like this:
+
+<!-- IMG -->
+
+Our files are now complete. Build the module with the latest changes:
+
+```
+npm run build
+```
+
+## Add Module to Directus
+
+When Directus starts, it will look in the `extensions` directory for any subdirectory starting with
+`directus-extension-`, and attempt to load them.
+
+To install an extension, copy the entire directory with all source code, the `package.json` file, and the `dist`
+directory into the Directus `extensions` directory. Make sure the directory with your extension has a name that starts
+with `directus-extension`. In this case, you may choose to use `directus-extension-module-landing-page`.
+
+Restart Directus to load the extension.
+
+:::info Required files
+
+Only the `package.json` and `dist` directory are required inside of your extension directory. However, adding the source
+code has no negative effect.
+
+:::
```

Comment:
I'm going to leave it in, because it's the same across all of these existing guides. However, when we update how extensions are loaded in, I'll audit them all and consolidate as appropriate.

Reply:
Perfect, thanks!